apps/civil/civil-sdt/demo-01.yaml

```diff
@@ -8,4 +8,5 @@ spec:
     java:
       image: hmctspublic.azurecr.io/civil/sdt:prod-61f79fc-20231002103124 #{"$imagepolicy": "flux-system:demo-civil-sdt"}
       environment:
+        PURGE_SCHEDULE_CRON: 0 15 0 * * 1,2
```

Comment:
Purge job needs to run 7 days a week.  Suggest splitting it up as 1, 2 and 3 on one instance and 4, 5, 6 and 7 on the other.

Reply:
done